pytorch_lightning/utilities/memory.py

```diff
@@ -53,7 +53,8 @@ def is_oom_error(exception):
 def is_cuda_out_of_memory(exception):
     return isinstance(exception, RuntimeError) \
         and len(exception.args) == 1 \
-        and "CUDA out of memory." in exception.args[0]
+        and "CUDA" in exception.args[0] \
+        and "out of memory" in exception.args[0]
```

Comment:
would be easier to read if the result will be in a var and then just return it...

Reply:
I think so, but was trying to as non-invasive as possible :)

Reply:
good invasion is always welcome :]